ibm/resource_ibm_is_vpc_routing_table_route.go

```diff
@@ -159,7 +159,7 @@ func resourceIBMISVPCRoutingTableRouteCreate(d *schema.ResourceData, meta interf
 		Address: core.StringPtr(nextHop),
```

Comment:
I think we need to handle both address and id ..something simialr to cli code
https://github.ibm.com/Bluemix/riaas-cli/blob/4c716cc0c529a4192961263ec7cf9ea501582b8d/plugin/commands/vpc/route-table-route.go#L333

Reply:
This issue is in progress in a separate branch.